draft-huitema-dprive-dnsoquic.md

```diff
@@ -69,53 +86,49 @@ TCP and provides more efficient error corrections than UDP. DNS over QUIC
 (DNS/QUIC) has privacy properties similar to DNS over TLS specified in RFC7858,
 and performance similar to classic DNS over UDP.
 
-{mainmatter}
+--- middle
 
 # Introduction
 
-Domain Name System (DNS) concepts are specified in [@!RFC1034].  This
-document presents a mapping of the DNS protocol [@!RFC1035] over QUIC
-transport [@!I-D.ietf-quic-transport] [@!I-D.ietf-quic-tls].  The
+Domain Name System (DNS) concepts are specified in {{!RFC1034}}.  This
+document presents a mapping of the DNS protocol {{!RFC1035}} over QUIC
+transport {{!I-D.ietf-quic-transport}} {{!I-D.ietf-quic-tls}}.  The
 goals of this mapping are:
 
 1.  Provide the same DNS privacy protection as DNS over TLS (DNS/TLS)
-    [@?RFC7858]. This includes an option for the client to 
+    {{?RFC7858}}. This includes an option for the client to 
     authenticate the server by means of an authentication domain
-    name [@!RFC8310].
+    name {{!RFC8310}}.
 
 2.  Provide an improved level of source address validation for DNS
-    servers compared to DNS/UDP [@!RFC1035].
+    servers compared to DNS/UDP {{!RFC1035}}.
 
 3.  Provide a transport that is not constrained by path MTU limitations on the 
     size of DNS responses it can send.
 
 4.  Explore the potential performance gains of using QUIC as a DNS
-    transport, versus other solutions like DNS over UDP (DNS/UDP) [@!RFC1035] or
-    DNS/TLS [@?RFC7858].
-
-5.  Participate in the definition of QUIC protocols and API, by
-    outlining a use case for QUIC different from HTTP over QUIC
-    [@?I-D.ietf-quic-http].
+    transport, versus other solutions like DNS over UDP (DNS/UDP) {{!RFC1035}} or
+    DNS/TLS {{?RFC7858].
 
 In order to achieve these goals, the focus of this document is limited
-to the "stub to recursive resolver" scenario also addressed by [@?RFC7858].
+to the "stub to recursive resolver" scenario also addressed by {{?RFC7858}}.
 That is, the protocol described here works for queries and responses between
 stub clients and recursive servers. The specific non-goals of this document are:
 
-1.  No attempt is made to support zone transfers [@?RFC5936], as these
+1.  No attempt is made to support zone transfers {{?RFC5936}}, as these
     are not relevant to the stub to recursive resolver scenario.
 
 2.  No attempt is made to evade potential blocking of DNS/QUIC
     traffic by middleboxes.
 
 Users interested in zone transfers should continue using TCP based
 solutions. Users interested in evading middleboxes should
```

Comment:
Suggest: "Users interested in zone transfers should continue using TCP based
 solutions and will also want to take note of work in progress to encrypt zone transfers using DoT {{I-D.ietf-dprive-xfr-over-tls}}. "

Reply:
OK.